Standards/scs-0004-w1-achieving-certification-implementation.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Implementation hints for achieving SCS-compatible certification"
+type: Supplement
+track: Global
+status: Draft
+supplements:
+  - scs-0004-v1-achieving-certification.md
+---
+
+## Process overview
+
+The *SCS-compatible* Certification for Operators is a technical certification:
+The operator needs to fulfill technical requirements, such as providing certain
+APIs and guaranteeing certain platform behavior in order to be certifiable.
+
+These requirements are meant to provide guarantees to their customers, allowing
+them to rely on certain features to be available and on certain system behavior
+that lets their applications run in a reliable way.
+
+The SCS certification process typically consists of a few simple steps:
+
+1. Running the SCS compliance test suite and adjusting the infrastructure until it passes.
+2. Any additional declarations (for non-testable aspects) are written and passed to the SCS certification body.
+3. The operator must be a member ("shaper" or "advisor" level) of the Forum SCS-Standards in the
+   OSB Alliance (a non-profit) and pay the respective membership fees. Alternatively fees can
+   be paid without becoming a member.
+4. The cloud can be listed on the SCS pages as *SCS-compatible* with a compatibility status that is
+   updated on a daily basis. SCS then tests the infrastructure on a daily basis.
+
+The precise rules that govern how certificates are issued or withdrawn are defined in the
+[SCS standard 0004](scs-0004-v1-achieving-certification.md).
+
+## Self-testing and technical adjustments
+
+In order for a cloud service offering to obtain a certificate, it has to
+conform to all standards of the respective scope, which will be tested at
```

Comment:
```suggestion
conform to all mandatory aspects of all standards of the respective scope, which will be tested at
```
Be more precise -- there can be *recommended* aspects and they are no prerequisite for passing.
We could go into more detail here and say that the recommendations should be taken seriously and providers should be prepared to answer customers why they don't fulfill these ...
But I preferred to keep it short here.